tests/automation_framework/data/work_order/work_order_submit.yaml

```diff
@@ -1083,75 +548,87 @@ test_workordersubmit_requestersignature_differentlength:
 test_workordersubmit_responsetimeout_string:
   responseTimeoutMSecs: str
 test_workordersubmit_sdk_dataencryptionalgorithm_list:
-  dataEncryptionAlgorithm: '["AES-GCM-256","AES-GCM-128","AES-GCM-625"]'
+  dataEncryptionAlgorithm:
+  - AES-GCM-256
+  - AES-GCM-128
+  - AES-GCM-625
 test_workordersubmit_sdk_dataencryptionalgorithm_listsamealgotwice:
-  dataEncryptionAlgorithm: '["AES-GCM-256","AES-GCM-256"]'
+  dataEncryptionAlgorithm:
+  - AES-GCM-256
+  - AES-GCM-256
 test_workordersubmit_sdk_workerEncryptionKey_special_character:
   workerEncryptionKey: hs$vi
 test_workordersubmit_sdk_workerencryptionkey_empty:
-  workerEncryptionKey: NONE
+  workerEncryptionKey: null
 test_workordersubmit_sessionkeyivInDataIv_hexstring:
-  iv: '"0x2345"'
+  iv: '0x2345'
 test_workordersubmit_sessionkeyiv_allspecial_characters:
   sessionKeyIv: \\@!$&*
 test_workordersubmit_workerEncryptionKey_notdefaulthex:
-  workerEncryptionKey: '234021262340212623402126'
+  workerEncryptionKey: 234021262340212623402126
 test_workordersubmit_workerEncryptionKey_special_character:
   workerEncryptionKey: hs$vi
 test_workordersubmit_workerId_notdefaultlength:
   workerId: 6ba1f459476bc43b65fd554f6b65910a8f551e4bcb0eee6a96dcebaeb14f2ae923456234564567
+test_workordersubmit_workerId_notdefaultlength_postmsg:
+  '"inData"':
```

Comment:
This keyword is still under single quotes and double quotes

Reply:
Fixed